### DISCUSSION TITLE
Can we add http interceptor like Alice for GetConnect?

### DISCUSSION THREAD

**Discussion by awaisabbasaa11** [jonataslaw/getx#1822] [Q&A]
I want to add a HTTP interceptor for example 

Alice interceptor
https://pub.dev/packages/http_interceptor

**Comment by moreirawebmaster** [comment]
Create uma proprerty with name interpector all requests eand response equals HttpClient and Gleap. I want using Gleap Interpector in you totality.
https://pub.dev/packages/gleap_http_interceptor
I know GetConnect have AddRequestModify and AddResponseModify, but not using complete interceptor freandly.

**Describe the solution you'd like**
Show Dio and Http your interpectors e apply in GetConnect.

**Describe alternatives you've considered**
Dio and HttpClient.
